gnosis/safe-react-e2e-tests
# Feature/check safe apps liveness

# What it solves
This PR resolves gnosis/safe-react-apps#162

# How this PR fixes it
- Adding a new suite for get the list of safes apps and check if they load properly
- We are going to search for HTML elements in the iframe or for errors loading the content. We are going as well to check for the SafeProvider to fail loading the app
- Reporting a success status even with failing apps. The objective is to generate a report and sending a continuous error status could be confusing
- Sending the report as a slack message
- Refactor folder structure to keep safe apps specific tests in their own directory

# How to test it
1. Execute the workflow locally or using github actions
2. Check is finishing with a success status
3. Check the problematic apps in the slack message
4. Validate the applications included in the message are not working properly
